public/app/features/dimensions/editors/ResourcePicker.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useState, ChangeEvent } from 'react';
+import {
+  TabContent,
+  Label,
+  Button,
+  Select,
+  Input,
+  Spinner,
+  TabsBar,
+  Tab,
+  StringValueEditor,
+  useTheme2,
+  stylesFactory,
+} from '@grafana/ui';
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { ResourceCards } from './ResourceCards';
+import SVG from 'react-inlinesvg';
+import { css } from '@emotion/css';
+
+interface Props {
+  value: string; //img/icons/unicons/0-plus.svg
+  onChange: (value?: string) => void;
```

Comment:
FYI -- drone was failing becasue the type error above:
![image](https://user-images.githubusercontent.com/705951/132042460-40abfc3c-7516-470d-8d8c-373c53e2bf0a.png)

previously it was a required string, not an optional string

Reply:
garwwww -- did not fix it.  hymmmm